src/components/Login/Step1/hooks.ts

```diff
@@ -115,6 +118,49 @@ const useMetamaskLogin = (
   return generateNonce;
 };
 
+interface IUserOnboardData {
+  data: any;
+}
+
+const useUserOnboardData = () => {
+  const accessToken = getAccessToken();
+  const user = useSelector((state: RootState) => state.user.user);
+  const navigate = useNavigate();
+  const onboardDataSave = useMutation(
+    async (data: IUserOnboardData) => {
+      const objectives = data.data[0].map(function (item: any) {

```

Comment:
What kind of structure this is having? Only mapping the 0th element.

Reply:
![image](https://user-images.githubusercontent.com/13116007/234449376-5ada682d-c0a5-43be-9d2d-6e7b3ad465a0.png)